src/cdk/a11y/a11y.md

```diff
@@ -55,6 +55,64 @@ interface Highlightable extends ListKeyManagerOption {
 
 Each item must also have an ID bound to the listbox's or menu's `aria-activedescendant`.
 
+### TreeKeyManager
+
+`TreeKeyManager` manages the active option in a tree view. This is intended to be used with
+components that correspond to a `role="tree"` pattern.
+
+#### Basic usage
+
+Any component that uses a `TreeKeyManager` will generally do three things:
```

Comment:
```suggestion
Any component that uses a `TreeKeyManager` should do three things:
```
nit: technical docs should already be present tense. Also replaced "generally do" with the stronger  "should" (which I think is accurate here)